app/Observers/RepositoryObserver.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Observers;
+
+use Exception;
+use App\Models\Webhook;
+use App\Models\Repository;
+use App\Actions\Github\Repository\CreateWebhook;
+use App\DTO\Github\Repository\Webhook\WebhookCreatedData;
+
+class RepositoryObserver
+{
+    /**
+     * Handle the Repository "created" event.
+     */
+    public function created(Repository $repository): void
+    {
+        try {
+            /** @var WebhookCreatedData $webhookData */
+            $webhookData = app(CreateWebhook::class)->execute($repository)->dtoOrFail();
+
+            $repository
+                ->webhooks()
+                ->save(
+                    new Webhook([
+                        'title'   => $repository->full_name . ' hook',
+                        'hook_id' => $webhookData->id,
+                    ]),
+                );
+        } catch (Exception $e) {
+            return;
```

Comment:
shall we do anything else in catch ?

Reply:
Yes but idk what to do for now, log ? Create without id and cron job to retry? Idk :(

Reply:
we can flag this kind of repository or PRs and fetch them manually in corn and queue. let's make an issue about it.

Reply:
https://github.com/geeksesi/code-review-pals/issues/60